packages/clients/meldemichel/API.md

```diff
@@ -72,101 +54,91 @@ A document rendering the map client could e.g. look like this:
+            movable: 'drag', // or 'none' to have an immovable pin
+            style: {
+              fill: '#0392cf' // or (optionally) '#ff0019' if 'none' is set
+            },
+          },
+        },
+      })
+
+      // If an old view state should be set, use this snippet:
+      meldemichelMapInstance.$store.dispatch('meldemichel/setMapState', {
+        vendor_maps_position: '566167.224436667,5935872.419250831',
+        vendor_maps_address_str: 'Alte Rabenstraße',
+        vendor_maps_address_hnr: '28',
+        mapZoomLevel: 6,
+        mapBaseLayer: 452,
+        mapCenter: '566808.8386735287,5935896.23173797',
+        // NOTE: vendor_maps_address_to and -_plz are not read
       })
 
-      // TODO example of subscribing to store values and putting them in the paragraphs above
+      // to retrieve map state updates, use this snippet:
+      mapInstance.$store.watch(
+        (_, getters) => getters["meldemichel/mapState"],
+        ({
+          mapCenter,
+          mapZoomLevel,
+          mapBaseLayer,
+          vendor_maps_position,
+          // The following fields are not changed/set in 'SINGLE' mode
+          vendor_maps_address_hnr,
+          vendor_maps_address_str,
+          vendor_maps_address_plz,
+          vendor_maps_address_to
+        }) => {
+          // do anything with the map values here; example print
+          console.info(`MapState Update
+            Center coordinate: ${mapCenter}
+            Zoom level: ${mapZoomLevel}
+            Base layer: ${mapBaseLayer}
+            Pin coordinate: ${vendor_maps_position}
+            Address: ${vendor_maps_address_str + ' ' + vendor_maps_address_hnr}
+            PLZ: ${vendor_maps_address_plz}
+            Distance to address: ${vendor_maps_address_to}`)
+        },
+        // will return initial values; delete parameter if not desired
+        { immediate: true }
+      )
     </script>
   </body>
 </html>
 ```
 
-### report
-
-// TODO translate after implementation (details might change)
-
-Der Reportmodus ist für die Verwendung im AfM gedacht. In diesem Modus fehlen alle bereits existierenden Meldungen, d.h. die Karte beinhält keine Schäden und keine dazugehörigen Elemente Filter, Meldungstoggler, Liste. Ebenso wird der Button "Neues Anliegen" nicht angezeigt. Dieser Modus wird mit dem Schlüsselwertpaar `mode: 'report'` in der `mapConf` hergestellt. Die Angabe der `reportLayerId` und des entsprechenden Eintrags in `overlays` und `attributions` muss in diesem Fall entfernt werden.
-
-Previously supplied map client values can be used to reconstruct a previous view.
+## Rendering COMPLETE mode
 
-```js
-import meldemichelMapClient from '@polara/client-meldemichel'
+The `index.html` included in the package's `dist` folder has been prepared for this mode and must merely be hosted.
 
-const mapInstance = await meldemichelMapClient.createMap({
-  // ... see `createMap` description
-})
+// TODO update this block after implementation (TO BE IMPLEMENTED)
```

Comment:
Shouldn't the paragraphs below already translated now?

Reply:
No, this is the "COMPLETE" region of the document. We're not done with the implementation here.